web3_proxy/src/rpcs/blockchain.rs

```diff
@@ -244,11 +244,11 @@ impl Web3Rpcs {
                     )
                     .await?;
 
-                let block = response.result.context("failed fetching block")?;
+                let block = response.result.web3_context("failed fetching block")?;
 
                 let block: Option<ArcBlock> = serde_json::from_str(block.get())?;
 
-                let block: ArcBlock = block.context("no block in the response")?;
+                let block: ArcBlock = block.web3_context("no block in the response")?;
```

Comment:
Does it make sense to differentiate between 5xx that originate on our (web3proxy) side, vs. 5xx that stemmed from third party applications?